### PR TITLE
Fixed documentation

### LINKED ISSUE #330: docs refer to constant that no longer exists
Two references to OG_MEMBERSHIP_TYPE_DEFAULT in docs, which looks like is now OgMembershipInterface::TYPE_DEFAULT

### PR DESCRIPTION
**Pull request by MPParsley**
Fix #330: OgMembershipInterface::TYPE_DEFAULT

### CODE REVIEW

File: src/MembershipManagerInterface.php [hunk 1]
```diff
@@ -132,7 +132,7 @@ public function getGroupMembershipsByRoleNames(EntityInterface $group, array $ro
    * @param \Drupal\Core\Session\AccountInterface $user
    *   The user object.
    * @param string $membership_type
-   *   (optional) The membership type. Defaults to OG_MEMBERSHIP_TYPE_DEFAULT.
+   *   (optional) The membership type. Defaults to OgMembershipInterface::TYPE_DEFAULT.
```
Comment (pfrenssen) [inline src/MembershipManagerInterface.php]:
```suggestion
   *   (optional) The membership type. Defaults to
       \Drupal\og\OgMembershipInterface::TYPE_DEFAULT.
```

File: src/Og.php [hunk 2]
```diff
@@ -176,7 +176,7 @@ public static function getMembership(EntityInterface $group, AccountInterface $u
    * @param \Drupal\Core\Session\AccountInterface $user
    *   The user object.
    * @param string $membership_type
-   *   (optional) The membership type. Defaults to OG_MEMBERSHIP_TYPE_DEFAULT.
+   *   (optional) The membership type. Defaults to OgMembershipInterface::TYPE_DEFAULT.
```
Comment (pfrenssen) [inline src/Og.php]:
```suggestion
   *   (optional) The membership type. Defaults to
       \Drupal\og\OgMembershipInterface::TYPE_DEFAULT.
```

**Review by pfrenssen — CHANGES_REQUESTED** [review]
These changes are failing coding standards checks because they exceed 80 characters.

File: src/MembershipManagerInterface.php [hunk 3]
```diff
@@ -132,7 +132,8 @@ public function getGroupMembershipsByRoleNames(EntityInterface $group, array $ro
    * @param \Drupal\Core\Session\AccountInterface $user
    *   The user object.
    * @param string $membership_type
-   *   (optional) The membership type. Defaults to OG_MEMBERSHIP_TYPE_DEFAULT.
+   *   (optional) The membership type. Defaults to
+       \Drupal\og\OgMembershipInterface::TYPE_DEFAULT.
```
Comment (MPParsley) [inline src/MembershipManagerInterface.php]:
```suggestion
   *   \Drupal\og\OgMembershipInterface::TYPE_DEFAULT.
```

File: src/Og.php [hunk 4]
```diff
@@ -176,7 +176,8 @@ public static function getMembership(EntityInterface $group, AccountInterface $u
    * @param \Drupal\Core\Session\AccountInterface $user
    *   The user object.
    * @param string $membership_type
-   *   (optional) The membership type. Defaults to OG_MEMBERSHIP_TYPE_DEFAULT.
+   *   (optional) The membership type. Defaults to
+       \Drupal\og\OgMembershipInterface::TYPE_DEFAULT.
```
Comment (MPParsley) [inline src/Og.php]:
```suggestion
   *   \Drupal\og\OgMembershipInterface::TYPE_DEFAULT.
```

**Comment by MPParsley** [comment]
Thanks, the 80 characters limit should now pass.

**Review by pfrenssen — APPROVED** [review]

**Pull request merged**